src/sbvr-api/sbvr-utils.ts

```diff
@@ -969,60 +969,62 @@ export const runURI = (
 		tx,
 	} as any;
 
-	return new Promise((resolve, reject) => {
-		const res: _express.Response = {
-			on: _.noop,
-			statusCode: 200,
-			status(statusCode: number) {
-				this.statusCode = statusCode;
-				return this;
-			},
-			sendStatus: (statusCode: number) => {
-				if (statusCode >= 400) {
-					const ErrorClass =
-						statusCodeToError[statusCode as keyof typeof statusCodeToError];
-					if (ErrorClass != null) {
-						reject(new ErrorClass());
+	return new Promise<PinejsClientCoreFactory.PromiseResultTypes>(
```

Comment:
The only actual change here is specifying the promise's generic type (adding the `<PinejsClientCoreFactory.PromiseResultTypes>`), the rest of the changes are due to the line going past a certain length so prettier reformatted the whole thing